### ISSUE TITLE
API for orders

### ISSUE THREAD

**Issue by JuggernautPanda** [WazirX/wazirx-api#11]
Is there no API documentation avaiable for orders?

It would be great to have one.

**Comment by rajasimon** [comment]
I am also looking for API for orders. As a developer I wanted to have a flexibility of orders and I think I can program one for myself. 

Is there anything wazirx planning to release.?

**Comment by sanket-dm** [comment]
Yeah, it would be great for you to release documentation on Buy and Sell orders, also on checking portfolio composition and balance.

**Comment by devakrishna33** [comment]
> I am also looking for API for orders. As a developer I wanted to have a flexibility of orders and I think I can program one for myself.
> 
> Is there anything wazirx planning to release.?

Is programming one for ourselves allowed?

**Comment by rajasimon** [comment]
@devakrishna33 So far all the exchanges I know this wazirx only doesn't allowed API access for orders. 

Yeah it's allowed. I don't have the specific numbers but it's pretty common.

I bet wazirx using binance exchange under the hood so it's matter of fact they flip the switch. I feel bad to for choosing wazirx but they have great btc/inr and depost INR is great too.
